src/mame/heathkit/tlb.cpp

```diff
@@ -141,26 +148,26 @@ device_heath_tlb_card_interface::device_heath_tlb_card_interface(const machine_c
 /**
  * original Heath H19 functionality
  */
-heath_tlb_device::heath_tlb_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock) :
+heath_tlb_device::heath_tlb_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock) :
 	heath_tlb_device(mconfig, HEATH_TLB, tag, owner, clock)
 {
 }
 
-heath_tlb_device::heath_tlb_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock) :
+heath_tlb_device::heath_tlb_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, u32 clock) :
 	device_t(mconfig, type, tag, owner, clock),
 	device_heath_tlb_card_interface(mconfig, *this),
-	m_maincpu(*this, "maincpu"),
-	m_screen(*this, "screen"),
-	m_palette(*this, "palette"),
-	m_crtc(*this, "crtc"),
-	m_p_videoram(*this, "videoram"),
-	m_p_chargen(*this, "chargen"),
-	m_config(*this, "CONFIG"),
-	m_ace(*this, "ins8250"),
-	m_beep(*this, "beeper"),
-	m_mm5740(*this, "mm5740"),
-	m_kbdrom(*this, "keyboard"),
-	m_kbspecial(*this, "MODIFIERS"),
+	m_maincpu(*this,      "maincpu"),
+	m_screen(*this,       "screen"),
+	m_palette(*this,      "palette"),
+	m_crtc(*this,         "crtc"),
+	m_p_videoram(*this,   "videoram"),
+	m_p_chargen(*this,    "chargen"),
+	m_config(*this,       "CONFIG"),
+	m_ace(*this,          "ins8250"),
+	m_beep(*this,         "beeper"),
+	m_mm5740(*this,       "mm5740"),
+	m_kbdrom(*this,       "keyboard"),
+	m_kbspecial(*this,    "MODIFIERS"),
 	m_repeat_clock(*this, "repeatclock")
```

Comment:
The trouble with tabulating everything like this is that if you introduce a data member with a longer name or remove the data member with the longest name, every line in the block needs to change.

Also, in this case you’ve tabulted the tags, but not the `*this` references.

Reply:
Ok, I undid these changes and the other ones you pointed out.